### ISSUE TITLE
Delekaq

### ISSUE THREAD

**Issue by Stahn456** [WarEmu/WarBugs#19467]
Delekaq the scorpion in Cinderfall used for the final quest All Da Pretty Beasties from K8P is not dropping his egg as he should. Also, the scorpions located in the cave in which you find him do not provide experience. I killed him twice and got no experience, waited for his respawn while my friend butchered almost 100 scorpions at level 33 and I at 38, got no experience at all. I tried to capture the combat log proving it, but apparently there is a bug that forces it to continue to scroll up/down even when nothing is being added too it. PITA to get a picture of it. It does however show that I was there and fought him and if you look on the right you can see the quest is not complete from the egg. Constantly spawning scorpions makes taking a proper photo extremely difficult there with the combat log issue I mentioned on top of it.

**Expected behavior and actual behavior:**
He should drop the egg from the quest as every other monster in the quest chain has done. Drops a random junk loot item instead.

**Steps to reproduce the problem:**

**Testing Screenshots/Videos/Evidences (always needed):**

![Consternation_004](https://user-images.githubusercontent.com/79117022/134848128-9d0205c3-79c9-4145-bbb2-da2077c7e7ec.jpg)

**Comment by lilimx** [comment]
Hey,

You and your friend have both the same quest?

**Comment by Stahn456** [comment]
No, only I had the quest, he was just there to heal me through the hero fights I needed.

Sent from Yahoo Mail on Android 
 
  On Mon, Sep 27, 2021 at 6:08 AM, ***@***.***> wrote:   


Hey,

You and your friend have both the same quest?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Stahn456** [comment]
Oh, and I did the majority of the looting. He was butchering and ran low on space and he never looted the hero until after I had. It simply didn't drop the egg. It dropped a leg segment and some other grey item. Didn't get a good look because of adds. Friend was named Mrfeelgood I think, it was an alt of Palerider his main. Mrfeelgood is a zealot if you wanted to look up the information on him and check the butchering thing out. He said he went from about 30 to over 115 when I asked at one point.

Sent from Yahoo Mail on Android 
 
  On Mon, Sep 27, 2021 at 9:08 AM, ***@***.******@***.***> wrote:   No, only I had the quest, he was just there to heal me through the hero fights I needed.

Sent from Yahoo Mail on Android 
 
  On Mon, Sep 27, 2021 at 6:08 AM, ***@***.***> wrote:   


Hey,

You and your friend have both the same quest?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by lilimx** [comment]
Hey,

could you try something. Go again to the champion and kill him. Don't be in a grp, your friend can heal you outside the grp.

**Comment by Stahn456** [comment]
It'll have to wait until tonight, we are in NA so we play around 8pm our time to about midnight or 1am. I'll message him that you want us to try again that way to see if it works and I'll get back to you once we do. Thanks for looking into this.

Sent from Yahoo Mail on Android 
 
  On Mon, Sep 27, 2021 at 10:45 AM, ***@***.***> wrote:   


Hey,

could you try something. Go again to the champion and kill him. Don't be in a grp, your friend can heal you outside the grp.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by lilimx** [comment]
_Edited messages for a better overwiev_

**Comment by Stahn456** [comment]
Confirmed the quest completed while my friend was NOT in the party, the egg did drop. I have a picture here of my combat log as proof the scorpions and the mobs near the cave aren't giving any exp as well. We got silver and butchering supplies, but zero experience. I set the chat to have experience shown, and as you can see, there was none gained.
![Consternation_005](https://user-images.githubusercontent.com/79117022/135017651-7b3de0b1-68e3-45f9-843e-92cee5a686c2.jpg)

**Comment by lilimx** [comment]
Hey,

thanks for testing it again :D
We will take a look into the EXP thing, if it's intended or not.

**Comment by Stahn456** [comment]
We discovered the PQ outside needed for the quest An Arm for an Arm also gave no exp for kills. Like 60 gnoblers, 4 Champs, and a hero with adds all gave no exp. Just influence.

Sent from Yahoo Mail on Android 
 
  On Tue, Sep 28, 2021 at 2:42 AM, ***@***.***> wrote:   


Hey,

thanks for testing it again :D
We will take a look into the EXP thing, if it's intended or not.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
